TargetArea target_area
AreaDivision area_division
